portal/src/main/scala/cromwell/pipeline/datastorage/dao/repository/ProjectRepository.scala

```diff
@@ -16,6 +16,7 @@ class ProjectRepository(pipelineDatabaseEngine: PipelineDatabaseEngine, projectE
 
   def addProject(project: Project): Future[ProjectId] = database.run(projectEntry.addProjectAction(project))
 
-  def deactivateById(projectId: ProjectId): Future[Int] = database.run(projectEntry.deactivateProjectById(projectId))
+  def deactivateProjectById(projectId: ProjectId): Future[Int] =
+    database.run(projectEntry.deactivateProjectById(projectId))
```

Comment:
please use the same naming in all files (deactivateProjectById looks good, but UserRepository still provides deactivateById def)

Reply:
done